examples/eks-cluster-with-windows-support/main.tf

```diff
@@ -100,7 +112,7 @@ module "aws_vpc" {
 # Example to consume aws-eks-accelerator-for-terraform module
 #---------------------------------------------------------------
 module "aws-eks-accelerator-for-terraform" {
-  source = "../.."
+  source = "git@github.com:aws-samples/aws-eks-accelerator-for-terraform.git?ref=v3.2.1"
```

Comment:
Is this tagging because of the defect? I think we need to remove the tagging otherwise we will end up modifying for every release

Reply:
Yes, there are pros and cons of this approach. If this example is guaranteed to be tested for each change (not just release changes, but for every PR merge), then this is not needed. On the other hand, this approach will give us time to test this example before updating the release tag used here.

Reply:
We can pin it for now. Once we have we the examples testing framework merged we can remove the pin.